crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -27,7 +27,7 @@ export interface ConversationConfiguration {
     admins?: Uint8Array[];
     ciphersuite?: Ciphersuite;
     keyRotationSpan?: number;
-    externalSenders: Uint8Array[];
+    externalSenders: Array<Uint8Array[]>;
```

Comment:
An array of array of buffers (= byte arrays)?
That seems confusing from an API standpoint, might also be incorrect?

Can we express it better?

Reply:
Yes for the moment it's an `Array<PublicKey>`, later on it will be an `Array<X509Certificate>`.  
In both cases it will be a `Vec<Vec<u8>>` received from DS that is just passed along to CoreCrypto which will do the parsing. Since this value will never be introspected by CoreLogic I think a byte array suits

Reply:
What I mean is that currently, `Array<Uint8Array[]>` translates to `Vec<Vec<Vec<u8>>>`

Because `Uint8Array` is a `Vec<u8>`, `Uint8Array[]` is a `Vec<Vec<u8>>` and `Array<Uint8Array[]>` is `Vec<Vec<Vec<u8>>>`